Classes/Common/MetsDocument.php

```diff
@@ -459,6 +460,13 @@ public function getMetadata($id, $cPid = 0)
             'out_of_print' => [],
             'rights_info' => [],
             'collection' => [],
+            'object_name' => [],
+            'object_alternative_names' => [],
+            'object_location' => [],
+            'geonames' => [],
+            'wikidata' => [],
+            'wikipedia' => [],
+            'object_reconstruction' => [],
```

Comment:
This array needs to contain only those metadata fields, which are strictly required by other plugins/modules. So `holder` seems to be a necessary addition, since it is referenced in `Classes/Format/Mods.php`, but all the other additions should be omitted.